tests/integration_tests/test_tiny_sectors_geocolor.py

```diff
@@ -0,0 +1,246 @@
+        "geocolor "
+        "$GEOIPS_TESTDATA_DIR/test_data_geocolor/data/goes16/20240920.1500/OR_ABI-L1b-RadF-M6C13*.nc"  # noqa: E501
+    ),
+    # Tiny sectors for himawari-9 3km night only
+    (
+        "$GEOIPS/tests/integration_tests/tiny_sectors/tiny_sectors_geostationary.sh "
+        "test_himawari9_eqc_3km_night_20240924T1500Z "
+        "ahi_hsd "
+        "Test-AHI-B13-Night-Only "
+        "geocolor "
+        "$GEOIPS_TESTDATA_DIR/test_data_geocolor/data/himawari9/20240924.1500/*.DAT"  # noqa: E501
+    ),
+]
+
+
+def setup_environment():
+    """
+    Set up necessary environment variables for integration tests.
+
+    Configures paths and package names for the GeoIPS core and its plugins by
+    setting environment variables required for the integration tests. Assumes
+    that 'GEOIPS_PACKAGES_DIR' is already set in the environment.
+
+    Notes
+    -----
+    The following environment variables are set:
+    - geoips_repopath
+    - geoips_pkgname
+    - repopath
+    - pkgname
+    """
+    # Setup base geoips environment
+    setup_geoips_environment()
+    # Setup current repo's environment
+    os.environ["repopath"] = os.path.realpath(
+        os.path.join(os.path.dirname(__file__), "..", "..")
+    )
+    os.environ["pkgname"] = "geocolor"
+
+
+@pytest.mark.optional
+@pytest.mark.sample_output
+@pytest.mark.tiny_sector_evaluation
+@pytest.mark.tiny_sector_global_example
+@pytest.mark.parametrize("script", global_example_integ_test_calls)
+def test_integ_global_example_script(base_setup: None, script: str):  # noqa: F811
+    """
+    Run integration test scripts by executing specified shell commands.
+
+    Parameters
+    ----------
+    script : str
+        Shell command to execute as part of the integration test. The command may
+        contain environment variables which will be expanded before execution.
+
+    Raises
+    ------
+    subprocess.CalledProcessError
+        If the shell command returns a non-zero exit status.
+    """
+    setup_environment()
+    run_script_with_bash(script)
+
+
+@pytest.mark.optional
+@pytest.mark.sample_output
+@pytest.mark.tiny_sector_evaluation
+@pytest.mark.tiny_sector_overlay
+@pytest.mark.parametrize("script", tiny_sector_overlay_integ_test_calls)
+def test_integ_tiny_sector_overlay_script(base_setup: None, script: str):  # noqa: F811
+    """
+    Run integration test scripts by executing specified shell commands.
+
+    Parameters
+    ----------
+    script : str
+        Shell command to execute as part of the integration test. The command may
+        contain environment variables which will be expanded before execution.
+
+    Raises
+    ------
+    subprocess.CalledProcessError
+        If the shell command returns a non-zero exit status.
+    """
+    setup_environment()
+    run_script_with_bash(script)
+
+
+@pytest.mark.integration
+@pytest.mark.tiny_sector_evaluation
+@pytest.mark.tiny_sector
+@pytest.mark.parametrize("script", tiny_sector_integ_test_calls)
```

Comment:
These tiny sector tests are NOT optional - they are run with the full set of integration tests (we want to make sure the tiny sectors we create actually work, so they are all regularly tested)